pkg/skaffold/version/version.go

```diff
@@ -55,10 +55,23 @@ var Get = func() *Info {
 	}
 }
 
+var SetClient = func(user string) {
+	client = user
+}
+
 func UserAgent() string {
 	return fmt.Sprintf("skaffold/%s/%s", platform, version)
 }
 
+// UserAgentWithClient add the `-user` flag value to the request header
```

Comment:
```suggestion
// UserAgentWithClient returns a string to be passed as the `--user` flag value to a request header
```

Reply:
Made the suggestion. 
I re-worded the line to clarify when to use `UserAgentWithClient` vs just `UserAgent`